Rakefile

```diff
@@ -327,7 +326,7 @@ CONFIRMATION
     end
 
   system("powershell -nologo -noprofile -executionpolicy unrestricted Import-Module .\\scripts\\publishpkg.psm1;Publish-ChefPkg #{ENV["publishsettings"]} \"\'#{subscription_name}\'\" #{publish_uri} #{definitionXmlFile} #{postOrPut}")
-
+  File.open("C:\\publish.txt", "a+") {|f| f.write("\n#{args.extension_version}\t#{Time.now}\t #{File.dirname(__FILE__)}") }
```

Comment:
@btm, Azure gives 15 spaces to publish extension but it does not provide any API to list the internal published versions. So if anyone published extension internally and forgot to delete then it will throw an error regarding space conflict. Hence, for internal tracking we are storing the published extension to the file.
To store that file, we can't use `temp` folder because the file will be required for future tracking also and `temp` will get deleted. Also
we can't store in `current working directory` because each of the team member publishes from different machines/directories. The file will become inconsistent.
For now I am removing this line, because the purpose of the PR is different and it should not be stuck because of this line. We'll discuss this further.